_test/bootstrap.sh

```diff
@@ -8,20 +8,20 @@ fi
 if [ ! -z "$CABALVER" ]; then
     export PATH=/opt/cabal/${CABALVER}/bin:$PATH
 fi
+
+# What directory is this script in?
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+deps=$(
+  cut -d: -f2 "$DIR/dependencies.txt" |
+  xargs echo |
+  tr " " "\n" |
+  sort |
+  uniq |
+  # Exclude packages included in GHC
+  grep -v 'base\|array\|containers\|time'
```

Comment:
I think this behavior is *by design*. When you ask for a simple package, it will try to install the newest one possible. If you supply multiple packages, it will try to satisfy all restrictions at the same time.

You should probably use a single `cabal install`.